unified-codes/libs/ui/src/lib/components/entity/molecules/EntityTableRow.tsx

```diff
@@ -8,14 +8,15 @@ export interface EntityTableRowProps {
   rowProps?: TableRowProps;
   cellProps?: TableCellProps;
   entity?: IEntity;
+  onEntitySelect: (entityCode: string) => void;
 }
 
 export type EntityTableRow = React.FunctionComponent<EntityTableRowProps>;
 
-export const EntityTableRow: EntityTableRow = ({ rowProps, cellProps, entity }) => {
+export const EntityTableRow: EntityTableRow = ({ rowProps, cellProps, entity, onEntitySelect }) => {
   const { code, description, type } = entity || {};
   return (
-    <TableRow {...rowProps}>
+    <TableRow {...rowProps} onClick={() => onEntitySelect(code ?? '')}>
```

Comment:
it's not likely ( or ever going to happen ) - perhaps don't do anything if the code is undefined?

Reply:
Hm, true. That would be better than redirecting to details with no code (if somehow it did happen).